lib/src/README.md

```diff
@@ -38,19 +21,11 @@ This is the main entry point for the user-  `pub run ffigen`.
 - Command-line options:
     - `--verbose`: Sets log level.
     - `--config`: Specifies a config file.
-- `ffigen.dart` will first check if a dynamic library already exists and is up to date. If not, it tries to auto-create it. If that fails, user must excplicitly call [setup.dart](#setup.dart).
+- `ffigen.dart` will first try to find dynamic library in default locations. If not. If that fails, user must excplicitly specify location in config under the key `llvm-lib`.
```

Comment:
"If not." is not a sentence, it does not have a verb.

Reply:
Ah, seems like I left that out by mistake. Fixed it.

>This shows structs by value is a nice dart:ffi feature to have! 

Absolutely 🎉